bluesky_queueserver/manager/start_manager.py

```diff
@@ -214,36 +222,42 @@ def start_manager():
         "'existing_plans_and_devices.yaml' is used.",
     )
     parser.add_argument(
-        "--user_group_permissions",
+        "--user-group-permissions",
         dest="user_group_permissions_path",
         type=str,
         help="Path to file that contains lists of plans and devices available to users. "
         "The path may be a relative path to the profile collection directory. "
         "If the path is a directory, then the default file name "
         "'user_group_permissions.yaml' is used.",
     )
-    parser.add_argument("--kafka_topic", type=str, help="The kafka topic to publish to.")
+    parser.add_argument("--kafka-topic", dest="kafka_topic", type=str, help="The kafka topic to publish to.")
```

Comment:
I believe that `dest` is unnecessary here and elsewhere. Argparse is smart enough to do the `-` -> `_` replacement by default.

Reply:
I added `dest` here for consistency with other arguments. It seems safer to define `dest`, because it decouples the parameter name (which could change in the future) and the variable name (which could remain the same even if parameter name is slightly modified).